### PR TITLE
Added script for locust caller

### PR DESCRIPTION
**Pull request by AhmedMahdy14**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -57,6 +57,7 @@ typer = "^0.4.0"
 orjson = "^3.6.7"
 tanbih-pipeline = {extras = ["redis"], version = "^0.12.4"}
 fastapi-utils = "^0.2.1"
+locust = "^2.13.0"
```
Comment (yifan) [inline pyproject.toml]: lets' move this to [dev-dependencies] instead

Comment (AhmedMahdy14) [reply pyproject.toml]: ok

**Comment by yifan** [comment]
looks good, you can merge it after addressing the dependecy comment

**Comment by yifan** [comment]
@AhmedMahdy14 looks good to me

**Pull request merged**